tests/apichecks/crds_test.go

```diff
@@ -526,6 +526,16 @@ func TestCRDFieldPresenceInUnstructured(t *testing.T) {
 					}
 				}
 
+				// Exclude output-only spec fields.
+				switch crd.Name {
+				case "sqlusers.sql.cnrm.cloud.google.com":
+					switch fieldPath {
+					case ".spec.passwordPolicy.status[].locked",
+						".spec.passwordPolicy.status[].passwordExpirationTime":
```

Comment:
I would prefer that we write this to a golden file, as opposed to keep adding each case here.

Reply:
Sure! Will refactor it in the PubSubSubscription PR.